plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -1,9 +1,18 @@
 const { chromium, expect } = require( '@playwright/test' );
 const fs = require( 'fs' );
+const {
+	ADMIN_USER,
+	ADMIN_PASSWORD,
+	CUSTOMER_USER,
+	CUSTOMER_PASSWORD,
+} = process.env;
+const adminUsername = ADMIN_USER ?? 'admin';
+const adminPassword = ADMIN_PASSWORD ?? 'password';
+const customerUsername = CUSTOMER_USER ?? 'customer';
+const customerPassword = CUSTOMER_PASSWORD ?? 'password';
```

Comment:
I added these variables so that the admin and customer logins here in the global setup will no longer be hard-coded, thus, making it work on any environment we point it to.